openedx/openedx-platform
# fix: Added feature toggle check for account deletion API (#32846)

## Description

Based on [this](https://discuss.openedx.org/t/solving-the-issue-for-disable-account-deletion-in-the-account-mfe/10684) forum discussion, I have incorporated support for a check feature toggle/flag (FEATURES["ENABLE_ACCOUNT_DELETION"]) in the account deletion API.


Useful information to include:
NA

## Supporting information

[wg-test-relese-discussion](https://github.com/openedx/wg-build-test-release/issues/189
)
## Testing instructions

Please disable the "ENABLE_ACCOUNT_DELETION" feature flag and attempt to delete your account. You should receive a forbidden.

## Deadline
NA

## Other information

NA
